lib/ansible/modules/crypto/luks_device.py

```diff
@@ -115,6 +115,16 @@
             - "Will only be used when I(device) and I(label) are not specified."
         type: str
         version_added: "2.10"
+    type:
+        description:
+            - "This option allow the user explicit define the format of LUKS
+              container that wants to work with. Options are 'luks1' or 'luks2'"
+            - "Will use per default 'luks2' if I(label) is specified, and the
```

Comment:
I think it is a lot better if the module does not allow `type=luks1` to be specified when `label` is specified. I would remove this part from the documentation, and add something like `This cannot be specified if I(type) is set to C(luks1).` to the `label` documentation, and add some code which errors out when `type=luks1` is combined with `label`.

Reply:
You are right, done.